src/metakb/cli.py

```diff
@@ -14,178 +14,384 @@
-    help=(
-        "Password to provide to application Neo4j database. Can also be provided via environment variable METAKB_DB_PASSWORD, which takes priority."
-    ),
+def check_normalizers(
+    db_url: str | None, normalizer: tuple[NormalizerName, ...]
+) -> None:
+    """Perform basic checks on DB health and table population for normalizers. Exits with
+    status code 1 if >= 1 DB schema is uninitialized or critical tables appear empty for one
+    or more of the concept normalizer services.
+
+    \b
+        $ metakb check-normalizers
+        $ echo $?
+        1  # indicates failure
+
+    To select specific normalizer services, provide one or more arguments:
+
+        $ metakb check-normalizers therapy disease
+
+    \f
+    :param db_url: URL endpoint for normalizer databases. Overrides defaults or env vars
+        for each normalizer service.
+    :param normalizer: tuple (possibly empty) of normalizer names to check
+    """  # noqa: D301
+    if not check_normalizer_health(db_url, normalizer):
+        _logger.warning("Normalizer check failed.")
+        click.get_current_context().exit(1)
+    _logger.info("Normalizer check passed.")
+
+
+@cli.command()
+@click.option("--db_url", help=_normalizer_db_url_description)
+@click.argument(
+    "normalizer",
+    type=click.Choice(list(NormalizerName), case_sensitive=False),
+    nargs=-1,
 )
+def update_normalizers(
+    db_url: str | None, normalizer: tuple[NormalizerName, ...]
+) -> None:
+    """Reload gene, disease, and therapy normalizer data.
+
+    Forces delete of each prior to fetching and loading new data. If errors are
+    encountered, attempts to complete updates of other normalizers before exiting.
+
+    Providing no arguments will attempt to update all three:
+
+        $ metakb update-normalizers
+
+    Providing individual normalizer names as arguments will only update those
+    normalizers:
+
+        $ metakb update-normalizers disease therapy
+
+    \f
+    :param db_url: URL endpoint of normalizers DynamoDB database. If not given, the
+        individual normalizers will revert to their own defaults.
+    :param normalizer: tuple (possibly empty) of normalizer names to update
+    """  # noqa: D301
+    success = True
+    update_params = [
+        (NormalizerName.DISEASE, DISEASE_AWS_ENV_VAR_NAME),
+        (NormalizerName.THERAPY, THERAPY_AWS_ENV_VAR_NAME),
+        (NormalizerName.GENE, GENE_AWS_ENV_VAR_NAME),
+    ]
+    if normalizer:
+        update_params = [p for p in update_params if p[0] in normalizer]
+    for name, aws_env_var_name in update_params:
+        if aws_env_var_name in environ:
+            msg = (
+                f"Updating the {name.value} AWS database from the MetaKB CLI is "
+                f"prohibited. Unset the environment variable:`{aws_env_var_name}` to "
+                "proceed."
+            )
+            _logger.error(msg)
+            click.echo(msg)
+            success = False
+            continue
```

Comment:
Once we're fully set up to update normalizers in the cloud, I wonder if there's some configuration we could add to the Dynamo table templates that only allow writes to come from the identity that's granted to the cloud update services. It'd be nice if we didn't have to write all of these checks into the code itself.